### DISCUSSION TITLE
Alpinejs cdn is down

### DISCUSSION THREAD

**Discussion by mahmudtopu3** [alpinejs/alpine#3512] [4. General]
https://unpkg.com/alpinejs is down

**Comment by SimoTod** [comment]
You do know that opening a bug because on the Alpine repo because a third party CDN is down will not make the resolution any faater, right?
Jokes aside unpkg is not a reliable cdn, the documentation instructions switched to jsdeliver a while ago but you are probably better off just hosting the script.

**Comment by mahmudtopu3** [comment]
Thanks, probably on alpinejs website the example showing unpkg can be changed too.

**Comment by ekwoka** [comment]
Ah the homepage still shows unpkg...since that's not in this repo we can't make a PR for it.
